.github/workflows/on-pullrequest-dockerfile.yml

```diff
@@ -9,9 +9,6 @@ on:
   push:
     branches:
       - main
-    paths:
```

Comment:
This work is being triggered only when the PR includes a change into the Dockerfiles or the action, the issue with this is that it can cause the analysis to go without running for a while, therefore, if there are new vulnerabilities it can go unnoticed.

I'm adding this run after merging the PRs to avoid adding noise to the PR itself since current behavior only shows the status, it does not forces or fails the build when new vulnerabilities appear.